roofit/hs3/src/RooJSONFactoryWSTool.cxx

```diff
@@ -1584,6 +1590,50 @@ bool RooJSONFactoryWSTool::importYML(std::string const &filename)
    return this->importYML(infile);
 }
 
+void RooJSONFactoryWSTool::importFunctionfromString(const std::string &jsonString){
+   std::stringstream is(jsonString);
+   this->importFunction(JSONTree::create(is)->rootnode(), false);
```

Comment:
```suggestion
   this->importFunction(JSONTree::create(jsonString)->rootnode(), false);
```
You should `git rebase` your three commits on the current ROOT master and do this change, because ROOT master now includes a commit where you can directly create a JSONTree from a string:
https://github.com/root-project/root/commit/392e2f77e618136238886dbacfa4496ace42c487

Same for `importVarfromString()`.